packages/tempus/src/Tempus_StepperOperatorSplit_decl.hpp

```diff
@@ -58,6 +61,19 @@ class StepperOperatorSplit : virtual public Tempus::Stepper<Scalar>
     int order,
     int orderMin,
     int orderMax);
+#endif
+
+  /// Constructor                                                                                               
+  StepperOperatorSplit(
+    std::vector<Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> > > appModels,
+    std::vector<Teuchos::RCP<Stepper<Scalar> > > subStepperList,
+    const Teuchos::RCP<StepperOperatorSplitAppAction<Scalar> >& stepperOSAppAction,
```

Comment:
To be consistent with the other steppers, we should probably make the stepperOSAppAction the last argument.  I did this because I have run into issues in the past were the compiler cannot tell what is wrapped inside the RCP, so it does not see the difference between the two argument signatures.

Reply:
I fixed this issue!